src/core/public/chrome/ui/header/header.tsx

```diff
@@ -443,22 +539,45 @@ export function Header({
 
       {/* Secondary header */}
       <EuiHeader className="newTopNavHeader">
-        <EuiHeaderSection side="left">
-          <EuiHeaderSectionItem border="none" data-test-subj="headerApplicationTitle">
-            <EuiTitle size="l" className="newTopNavHeaderTitle">
-              {breadcrumbs && <h1>{breadcrumbs[breadcrumbs.length - 1]?.text}</h1>}
-            </EuiTitle>
-          </EuiHeaderSectionItem>
-
-          {renderBadge()}
-          {renderLeftControls()}
-        </EuiHeaderSection>
-
-        <EuiHeaderSection side="right">
-          {renderCenterControls()}
-          {renderActionMenu()}
-          {renderRightControls()}
-        </EuiHeaderSection>
+        <EuiFlexGroup
+          justifyContent="spaceBetween"
+          gutterSize="s"
+          className="secondaryPageHeaderFlexGroup"
+        >
+          {/* Left Section */}
+          <EuiHeaderSection side="left">
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <EuiHeaderSectionItem border="none" data-test-subj="headerApplicationTitle">
+                  <EuiTitle size="l" className="newTopNavHeaderTitle">
+                    {breadcrumbs && <h1>{breadcrumbs[breadcrumbs.length - 1]?.text}</h1>}
+                  </EuiTitle>
+                </EuiHeaderSectionItem>
+              </EuiFlexItem>
+
+              {renderBadge() && <EuiFlexItem grow={false}>{renderBadge()}</EuiFlexItem>}
```

Comment:
instead of evaluating these twice, can we assign to a variable and reuse?

Reply:
yeah will do,